research/src/pages/select/select.proposal.mdx

```diff
@@ -195,11 +195,19 @@ and is therefore considered to be invalid.
 - The currently selected `<option>`, or the first if one isn't selected, should be
   visible within the listbox. This may require moving the list to accomodate listbox positioning
   and available space.
-- The user can dismiss the listbox and remove the state of `open` from the `<select>` in any of the
+- The user can dismiss the listbox and remove the state of `open` from the `<select>` in either of the
   following manners:
-  - The user clicks outside of the listbox
-  - Presses the escape key
-  - Selects one, or more (if `multiple` is true), by clicking or hitting `enter` or `space` key
+  - Triggering any of the listbox's <a href="#light-dismiss">light dismiss</a> behaviors
+  - Selecting one, or more (if `multiple` is true), options by clicking or hitting the `enter` or `space` key
+
+### Light dismiss <a class="link" href="#light-dismiss" id="light-dismiss"></a>
```

Comment:
Can we add an accessible name to the anchor link? Project convention seems to be an `aria-label="Anchor"`. I think we could iterate on that so that all the links have unique identifiers, but that's out of scope for this PR :)

Reply:
That link is actually not visible on the page (since it's empty); it's just there to be a target for # anchor navigation.  The 
The ![image](https://user-images.githubusercontent.com/1688716/88120539-fdedfa80-cb77-11ea-91d5-7f710fa2101a.png) link to the left of the header text is apparently generated by the build, with `aria-label="Anchor"` included:
![image](https://user-images.githubusercontent.com/1688716/88120381-9e8fea80-cb77-11ea-9820-6cc39ebd72a1.png)

However it turns out that I don't actually need that empty link at all.  The `<h3>` gets the right ID if I remove it, so anchor navigation to the section still works correctly.

![image](https://user-images.githubusercontent.com/1688716/88120680-5b824700-cb78-11ea-816f-c75922b9fe9f.png)

Reply:
(so I removed it).